src/System.Text.Json/tests/Serialization/BlogPost.cs

```diff
@@ -0,0 +1,364 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace System.Text.Json.Serialization.Tests.Schemas.BlogPost
```

Comment:
It was like that at first, updated because class names declared here very generic terms like `Post`, 'Member' can be overlapped easily https://github.com/dotnet/corefx/pull/39668#discussion_r306045839

Reply:
Ah I see. That makes sense.